.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: node_js
+node_js:
+  - stable
+  - 8.9.4
+
+services:
+  - postgresql
+
+cache:
+directories:
+  - "node_modules"
+
+
+before_script:
+  - psql -c 'create database travis_ci_test;' -U postgres
+
```

Comment:
I will suggest you use a database named `travis` instead of `travis_ci_test`

Reply:
I will correct this. Thanks